linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env groovy
 
+def FuncTestsFailedOnAzure
```

Comment:
you are using these as env variables, please define and use them as env._ENV_VARIABLE_NAME to be consistent with the Jenkins pipeline implementation. use the environment code block to define them.